### DISCUSSION TITLE
py.test: error: unrecognized arguments

### DISCUSSION THREAD

**Discussion by Dineshr08** [pytest-dev/pytest#11798] [Q&A]
From Jenkins free style project, In build section using execute windows batch command I am running below code. What i am trying to do is I am running python script, from that i am fetch output and try to pass to pytest cli.But i am getting error py.test: error: unrecognized arguments

Error: py.test: error: unrecognized arguments: --env_name=prod --testsuiteID=795328
@echo off
cd Jenkins_TestSuite
REM Execute the Python script and capture the output
for /f "delims=" %%i in ('python.exe Jenkins_TE_Updated.py "%TestCycle_Name%" %env_name%') do set RESULT=%%i
REM Display the result in the console output
echo Result: %RESULT%
set PYTEST_ARGS=--env_name="%env_name%" --html=Prod_Report.html -m "%test_type%" --testsuiteID="%RESULT%"
py.test Tests %PYTEST_ARGS%

**Comment by The-Compiler** [comment]
Those arguments indeed don't exist in pytest at all. Chances are you don't have whatever plugin installed that provides them.
